arrow-libs/gradle/libs.versions.toml

```diff
@@ -0,0 +1,59 @@
+[versions]
```

Comment:
Will this work with renovate like this? 😍

Reply:
Yeah, it will work, I have to cleanup how we are using all dependencies yet, but when I can delete all gradle.properties versions and move to the toml file, I will add the Renovate file :)

Reply:
Awesome. Really looking forward to seeing that in action.